packages/gatsby-source-wordpress/docs/plugin-options.md

```diff
@@ -1107,9 +1107,9 @@ Determines whether or not this type will be treated as an interface comprised en
 
 #### type.\_\_all.beforeChangeNode
 
-A function which is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it.
+A function which is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it. This option should be a path to a JS file where the default export is the beforeChangeNode function. The path can be relative to your gatsby-node.js or absolute. Currently you can inline a function by writing it out directly in this option but starting from Gatsby v4 only a path to a function file will work.
```

Comment:
This makes it sounds like they can use this version of the plugin and use a function or use the filepath, but if they use Gatsby v4 only the filepath will work. This is not true right? Even if they're using V3 but on this version of the plugin they will need a filepath if I'm understanding right.

```suggestion
A relative (to your `gatsby-node.js` file) or absolute path to a file where the default export is the `beforeChangeNode` function. This function is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it. 

NOTE: Previous versions let you inline a function by writing it out directly in this option.
```